### PR TITLE
Add consensus

### PR DESCRIPTION
**Pull request by luisas**
Add computation of consensus for MSA

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -108,7 +108,7 @@
             ext.prefix = { "${meta.id}" }
         }
 
-        withName: "CLUSTALO_ALIGN|FAMSA_ALIGN|LEARNMSA_ALIGN|MAFFT|MAGUS_ALIGN|MUSCLE5_SUPER5|REGRESSIVE|TCOFFEE_ALIGN|TCOFFEE3D_ALIGN|FOLDMASON_EASYMSA"{
+        withName: "CLUSTALO_ALIGN|FAMSA_ALIGN|KALIGN_ALIGN|LEARNMSA_ALIGN|MAFFT|MAGUS_ALIGN|MUSCLE5_SUPER5|REGRESSIVE|TCOFFEE_ALIGN|TCOFFEE3D_ALIGN|FOLDMASON_EASYMSA"{
```
Comment (itrujnara) [inline conf/modules.config:111]: Why is foldmason out of alphabetical order?

File: docs/images/nf-core-msa_metro_map.png
Comment (itrujnara) [inline docs/images/nf-core-msa_metro_map.png:1]: Full support for dark mode, the logo disappeared somewhere though

File: nextflow_schema.json [hunk 2]
```diff
@@ -60,6 +60,19 @@
                 }
             }
         },
+        "align_options": {
+            "title": "Stats options",
+            "type": "object",
+            "fa_icon": "fas fa-terminal",
+            "description": "Define wether to run stats.",
```
Comment (itrujnara) [inline nextflow_schema.json:67]: <pedantism> whether </pedantism>

**Review by itrujnara — APPROVED** [review]
The code is fine, some cosmetic comments added

File: conf/modules.config (hunk 1 above)
Comment (luisas) [reply conf/modules.config]: fixed

**Pull request merged**